src/visitor.rs

```diff
@@ -402,6 +402,8 @@ impl Visitor {
             }
             Type::Generic(_) => {}
             Type::Primitive(_) => {}
+            // Pattern types are unstable and rustc internal rust-lang#120131
+            Type::Pat { .. } => {}
```

Comment:
So far we've been supporting unstable compiler features in the event they stabilize. I'm fine with not supporting pattern types, but ideally it would panic rather than silently ignore the inner type.

Reply:
That makes sense. I've added a panic :+1: 
https://github.com/awslabs/cargo-check-external-types/compare/47b98dd4eea02026f5eb4e71142752350e6234d2..fb36e66e295c331089a2a957ac9c65eb5a65209f